cmd/info.go

```diff
@@ -90,30 +89,46 @@ func (o *InfoOptions) Run() error {
 		}
 	}
 
+	if len(o.Refs) < 1 {
```

Comment:
Just a small nit, but please make this `== 0` instead of `< 1`, which reads a bit clearer.